### DISCUSSION TITLE
mermaid-editor没有展示绘制结果

### DISCUSSION THREAD

**Discussion by ctyytc** [he3-app/he3-feedback#347] [Bug Report]
### Version
mermaid-editor 1.1.0

### Environment
Mac v2.0.17

### Steps to reproduce
1. Mac桌面端，打开mermaid-editor，自动同步开启
2. 浏览器，打开“ https://portal.he3app.com/home/extension/mermaid-editor ”，自动同步开启

### What is expected?
1. 看到自动绘制的图示，并能保持同步修改
2. 看到自动绘制的图示，并能保持同步修改

### What is actually happening?
1.  无法看到自动绘制的图示
2.  看到自动绘制的图示，并能保持同步修改
![image_2023-12-30_082805599](https://github.com/he3-app/he3-feedback/assets/413886/dd0ab736-ebc2-4bf9-be24-c442196d8b3b)
![image_2023-12-30_082648597](https://github.com/he3-app/he3-feedback/assets/413886/2aa2f7cc-db34-418b-bd82-39b14d2a1ca6)

**Comment by zhenweiwang1990** [comment]
Bug @yinjg1997
